src/hb-ot-os2-table.hh

```diff
@@ -155,24 +226,11 @@ struct OS2
   HBINT16	sTypoLineGap;
   HBUINT16	usWinAscent;
   HBUINT16	usWinDescent;
-
-  /* Version 1 */
-  //HBUINT32	ulCodePageRange1;
-  //HBUINT32	ulCodePageRange2;
-
-  /* Version 2 */
-  //HBINT16	sxHeight;
-  //HBINT16	sCapHeight;
-  //HBUINT16	usDefaultChar;
-  //HBUINT16	usBreakChar;
-  //HBUINT16	usMaxContext;
-
-  /* Version 5 */
-  //HBUINT16	usLowerOpticalPointSize;
-  //HBUINT16	usUpperOpticalPointSize;
-
+  OS2V1Tail	v1Tail;
+  OS2V2Tail	v2Tail;
+  OS2V5Tail	v5Tail;
```

Comment:
Please rename these `v1Tail` etc to `v1X`.  The X signifies that they are unsafe to access without restriction.  Then rename the `get_v1` etc to just `v1`.

Reply:
Done